chapter3/chapter3.tex

```diff
@@ -501,7 +502,7 @@ \section{Zwischenwertsatz}
     \item Wir zeigen erst durch Widerspruch, dass $f(s) \geq 0$.
       Falls $f(s) < 0$, setze
       $\varepsilon = |f(s)| > 0$. Da $f$ stetig in $s$ ist,
-      existiert $\delta > 0$ so dass
+      existiert $\delta > 0$, so dass
```

Comment:
Kommt das Komma in gewissen Situationen nicht nach dem "so"?

Reply:
Ja, insgesamt recht verwirrend. Ich schaue es bei der nächsten Gelegenheit mal im Wahrig nach.

Reply:
Nur wenn es dich auch interessiert :)